sys/kern/vmem.c

```diff
@@ -65,7 +86,7 @@ static vmem_freelist_t *bt_freehead_tofree(vmem_t *vm, vmem_size_t size) {
  * For VMEM_INSTANTFIT, returns the list in which any blocks are large enough.
```

Comment:
I guess that comment is not relevant anymore?

Reply:
Yes, I've deleted part of this comment: https://github.com/cahirwpz/mimiker/pull/593/commits/5e62fb32623934c1f8408ab62e709f7b7fa8a4e5